fluster/fluster.py

```diff
@@ -278,6 +278,12 @@ def run_test_suites(self, ctx: Context) -> None:
                         decoder.multiple_layers = True
                 if decoder.codec != test_suite.codec:
                     continue
+                if (
+                    test_suite.test_method is not None
```

Comment:
test_method sould never be none, it should be initialized to TestMethod.MD5 for those test suites that don't define it.